planning/behavior_velocity_planner/autoware_behavior_velocity_blind_spot_module/config/blind_spot.param.yaml

```diff
@@ -1,14 +1,18 @@
 /**:
   ros__parameters:
     blind_spot:
-      use_pass_judge_line: true
-      stop_line_margin: 1.0 # [m]
-      backward_detection_length: 100.0 # [m]
-      ignore_width_from_center_line: 0.7 # [m]
-      adjacent_extend_width: 1.5 # [m]
-      opposite_adjacent_extend_width: 1.5 # [m]
-      max_future_movement_time: 10.0 # [second]
-      ttc_min: -5.0 # [s]
-      ttc_max: 5.0 # [s]
-      ttc_ego_minimal_velocity: 5.0 # [m/s]
+      backward_attention_length: 100.0 # [m]
```

Comment:
Wouldn't it be better from a tuning perspective to calculate this parameter internally using the maximum speed of the considered object, the maximum velocity of the ego vehicle, and the prediction horizon for TTC as parameters?

Reply:
Since it's a niche topic, it's fine to address it later, but do you think there would be less room for tuning if we implemented the method/model used by the D&T team and allowed users to specify only intuitive values such as vehicle speed?